### ISSUE TITLE
Staurastrum mucronatum Ralfs ex Ralfs

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#2883]
**Staurastrum mucronatum Ralfs ex Ralfs**

The accepted name should be Staurodesmus mucronatus (Nägeli) Thomasson, 1955 according to WoRMS
(http://www.marinespecies.org/aphia.php?p=taxdetails&id=661690)

-----

Github user: @lmoliner
User: [See in registry](https://www.gbif.org/api/feedback/user/ecdadfbe5970e95772f2d27a4dd6a5c1:718f5a74b7801001eae4a2841c3a3483b8fe5f6f7ee4bb7c41bb5df9974bc14469f647e064dbf7ecefa31136e5c3dd12cbcdd839641f1eb1e4ecfb5e672b8f87)
System: Opera 69.0.3686 / Windows 8.1.0.0
Referer: https://www.gbif.org/species/2648226
Window size: width 1536 - height 726
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-07-19T16:59:56.802Z',mode:absolute,to:'2020-07-19T17:05:56.802Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2020-07-19T16:59:56.802Z',mode:absolute,to:'2020-07-19T17:05:56.802Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: OPERATIONAL

**Comment by jhnwllr** [comment]
contact dyntaxa

**Comment by jhnwllr** [comment]
**Staurastrum mucronatum Ralfs ex Ralfs**
https://www.gbif.org/species/2648226

should be 

**Staurodesmus mucronatus (Nägeli) Thomasson, 1955**

**Comment by lmoliner** [comment]
I agree with you, the most reliable source for phytoplankton taxonomy is AlgaeBase

________________________________
